sdcm/cluster.py

```diff
@@ -1839,6 +1840,15 @@ def config_setup(self,
 
             if ldap:
                 scylla_yml.update(self.get_ldap_config())
+            elif ms_ad_ldap:
+                scylla_yml.update({
+                    'ldap_attr_role': 'cn',
+                    'ldap_bind_dn': 'cn=admin,dc=scylla-qa,dc=com',
+                    'ldap_bind_passwd': 'scylla-qa2021!',
+                    'ldap_url_template':
+                        'ldap://54.198.162.249:389/dc=scylla-qa,dc=com?cn?sub?(member=CN={USER},DC=scylla-qa,DC=com)',
```

Comment:
I will also use the ldap address/port in saslauthd.conf, can you set it as a variable or macro?

Reply:
the address was added to `KeyStore` and the port is default